### PR TITLE
[SYCL] Fix wrong argument size in getImageInfo call

### PR DESCRIPTION
**Pull request by againull**
Signed-off-by: Artur Gainullin <artur.gainullin@intel.com>

### CODE REVIEW

File: sycl/include/CL/sycl/detail/image_impl.hpp [hunk 1]
```diff
@@ -239,7 +239,10 @@ class image_impl final : public SYCLMemObjT<AllocatorT> {
     MType = detail::convertChannelType(Format.image_channel_data_type);
     MNumChannels = getImageNumberChannels(MOrder);
 
-    getImageInfo(PI_IMAGE_INFO_ELEMENT_SIZE, MElementSize);
+    size_t ElementSize;
+    getImageInfo(PI_IMAGE_INFO_ELEMENT_SIZE, ElementSize);
+    assert(ElementSize <= 16);
+    MElementSize = static_cast<uint8_t>(ElementSize);
```
Comment (alexeyvoronov) [inline sycl/include/CL/sycl/detail/image_impl.hpp]: Maybe it's better to change the type of MElementSize to size_t?

**Comment by vladimirlaz** [comment]
Please rebase to top of sycl branch to let LIT test on windows pass.

File: sycl/include/CL/sycl/detail/image_impl.hpp (hunk 1 above)
Comment (alexeyvoronov) [reply sycl/include/CL/sycl/detail/image_impl.hpp]: Looks like a workaround.

Comment (againull) [reply sycl/include/CL/sycl/detail/image_impl.hpp]: Fixed.

**Pull request merged**